starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -49,4 +49,27 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+  
+  /**
+   * Test Create method 
+   */
+  @Test
+  public void create() {
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
```

Comment:
We would also like to set the inventory's id before `Inventory actualInventoryItem = this.inventoryDAO.create(inventory);` so we can test for different inventory ids, proving our create method works.

Set the `inventory`'s id similar to `inventory.setName` and `inventory.setProductType` with a constant (see line 27/28 as an example).

Reply:
Implemented in commit beee9ac62fec7cdbca505d54a12b1681a8a629eb